.github/workflows/bench-pr-comment.yml

```diff
@@ -32,14 +32,13 @@ jobs:
         run: sudo apt-get install -y pkg-config libssl-dev
       - uses: actions-rs/toolchain@v1
       - uses: Swatinem/rust-cache@v2
-      - name: Load env vars
+      - name: Set output type
         run: |
-          set -a
-          source bench.env
-          set +a
           echo "LURK_BENCH_OUTPUT=pr-comment" >> $GITHUB_ENV
-          env | grep -E 'LURK|EC_GPU|CUDA'
-        working-directory: ${{ github.workspace }}/benches
+      - uses: cardinalby/export-env-action@v2
```

Comment:
This is an unrelated fix that was previously made to the `gpu-benchmark` job below.